src/components/templates/Audience/Audience.tsx

```diff
@@ -47,6 +47,10 @@ const VIDEO_MIN_WIDTH_IN_SEATS = 8;
 // We should keep the 16/9 ratio
 const VIDEO_MIN_HEIGHT_IN_SEATS = VIDEO_MIN_WIDTH_IN_SEATS * (9 / 16);
 
+// Always have an odd number of rows and columns (because of the firelane delimiter).
+const DEFAULT_COLUMNS_NUMBER = 25;
+const DEFAULT_ROWS_NUMBER = 19;
```

Comment:
Audience is a bit different from Auditorium & is about to become legacy. I don't remember the exact reason, but they were explicitly decided to be left in the file scope.

Reply:
> they were explicitly decided to be left in the file scope.

okay, I see.

@mike-lvov, and what about this comment (in case you've missed / forgot)? https://github.com/sparkletown/sparkle/pull/1251#discussion_r664934885

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/1251/commits/0624acd900aecb7e3267234f3a30e76171589792